app/gateway/2.8.x/reference/configuration.md

```diff
@@ -143,9 +143,9 @@ As always, be mindful of your shell's quoting rules specifying values
 containing spaces.
 
 For more details on the Nginx configuration file structure and block
-directives, see https://nginx.org/en/docs/beginners_guide.html#conf_structure.
+directives, see the [official documentation](https://nginx.org/en/docs/beginners_guide.html#conf_structure).
```

Comment:
```suggestion
directives, see the [Nginx reference](https://nginx.org/en/docs/beginners_guide.html#conf_structure).
```
Slightly more descriptive text. Ideally, you want the link text to tell the reader what they're clicking on - especially important for accessibility/someone tabbing through the page. I think it's useful for SEO also.